services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -440,6 +440,7 @@ protected void initAttesterSlashingPool() {
                     slashing -> slashing.getIntersectingValidatorIndices().size())
                 .reversed());
     blockImporter.subscribeToVerifiedBlockAttesterSlashings(attesterSlashingPool::removeAll);
+    attesterSlashingPool.subscribeOperationAdded(forkChoice::onAttesterSlashing);
```

Comment:
This will only pick up slashings received via gossip but we'll miss any attestations included in blocks that we haven't already seen.

Reply:
Yeah, I've realized it yesterday, while getting to sleep. Will fix it now